app/assets/javascripts/directives/views/actionsMenu.ts

```diff
@@ -32,7 +32,7 @@ class ActionsMenuCtrl extends PureViewCtrl implements ActionsMenuScope {
 
   application!: WebApplication
   item!: SNItem
-  public loadingExtensions: boolean = true
+  loadingExtensions: boolean = true
```

Comment:
This might be removed per comment below, but I believe this can be simplified to `loadingExtensions = true`